sovrin_common/roles.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum, unique
+
+from plenum.common.roles import Roles
+
+
+@unique
+class Roles(Enum):
+    #  These numeric constants CANNOT be changed once they have been used,
+    #  because that would break backwards compatibility with the ledger
+    TRUSTEE = Roles.TRUSTEE.value
+    TGB = "1"
+    STEWARD = Roles.STEWARD.value
+    TRUST_ANCHOR = "3"
```

Comment:
Added a comment.
Also I made a change to use values starting from 100 for Sovrin roles, so we reduce a risk of overlap.